source/controllers/include/controllers/impedance/NewImpedance.hpp

```diff
@@ -0,0 +1,202 @@
+#pragma once
+
+#include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/Dense>
```

Comment:
Isn't that enough? And doesn't dense already include core?
```suggestion
#include <Eigen/Core>
#include <Eigen/Dense>
```

Reply:
No, all of control_libraries currently uses the `eigen3/Eigen` for includes. `eigen3` is installed on the path and contains `Eigen` inside. We have the line `include_directories(${Eigen3_INCLUDE_DIRS})` in the top-level CMakeLists but it doesn't do anything since that variable is empty (same with `${OsqpEigen_INCLUDE_DIR}`). That's why in some projects you see an invocation to `include_directories(/usr/local/include/eigen3)`, so that `#include <Eigen/...>` resolves directly

Reply:
Yes good point about the empty variables in the CMakeLists, I want to remove them at some point. Thanks for the clarification